### ISSUE TITLE
Allow different modes on resize

### ISSUE THREAD

**Issue by domenic** [bwinton/TabCenter#1036]
I have a Windows 10 tablet that I often rotate between landscape and portrait orientations. In landscape, I prefer "shrink sidebar when not in use", or maybe even "move tabs to top" (still experimenting). In portrait, I prefer the full experience with the sidebar always there.

I have a similar experience when I want to view things side by side in two windows, using e.g. Windows Key + Left Arrow. I'd like to use one of the more compact configurations then. I imagine anyone who often re

Basically, whenever the window is "too small", I'd like the add-on to automatically switch to a different mode.

I recognize that this might be a difficult UX to design good defaults for, and so far this add-on doesn't seem to be very configurable, so I'm not sure what the right solution is. But I can hope, and file issues!
